persistent-test/src/UpsertTest.hs

```diff
@@ -165,3 +165,48 @@ specsWith runDb handleNull handleKey = describe "UpsertTests" $ do
               Just 2
           Don'tUpdateNull ->
               Nothing
+
+  describe "putMany" $ do
```

Comment:
Moved from the `PersistentTest` module so that `mongoDB` could skip them for now.